### DISCUSSION TITLE
Form handleSubmit not seeing values when submit is fired from outside of the form in modal (context) footer

### DISCUSSION THREAD

**Discussion by j-lee8** [react-hook-form/react-hook-form#7870] [Q&A]
FYI it works fine when I have the submit button inside the form. I don't want this on mobile though.

On mobile I have a filter button that opens a Chakra UI modal (context) which has the form inside it as a child. So far so good.

The submit button is also passed to the context to be displayed in the modal, BUT the submit button is displayed in the modal's footer and not its children (where the form is).

E.g.,

```
showModal({
    isOpen: true,
    children: <FilterForm id="filter-form" />,
    footer: <SubmitButton form="filter-form">
                      Show results
                </SubmitButton>
});
```
Modal.tsx
```

<Modal>
    <ModalBody>
        {children}
    </ModalBody>
    <ModalFooter>
        {footer}
    </ModalFooter>
</Modal>
```

The handleSubmit inside the form fires, but the values are empty. I'm at a complete loss as to why this is.

The form and id fields match up and the submit does fire, but the values are empty. I really don't understand why this could be.

**Comment by Moshyfawn** [comment]
Please, consider providing a small codesanbox reproduction of the issue. We appreciate your issue description, but we can't be spending time trying to reproduce your environment which might not even be what you have

**Reply by j-lee8** [reply]
I went with adding the submit button inside the form so it works fine. It's not what I wanted but it'll suffice. If I find time I'll create a code sandbox.  Thank you.
